config/locales/en.yml

```diff
@@ -1098,6 +1103,8 @@ en:
       deployment_option_plugin: |
         Plugins provide a wrapper for the 3scale API that enables API Access Control and API Traffic Reporting without having to run another server.
         A plugin lives inside the code that powers your API. The plugin calls the 3scale API for Access Control & Traffic Reporting.
+      deployment_option_service_mesh: |
+        Service Mesh ...
```

Comment:
```suggestion
        The term service mesh is used to describe the network of microservices that make up the applications and the interactions between them. As a service mesh grows in size and complexity, it can become harder to understand and manage. Its requirements can include discovery, load balancing, failure recovery, metrics, and monitoring. A service mesh also often has more complex operational requirements, like A/B testing, canary releases, rate limiting, access control, and end-to-end authentication.
```